pages/tabs/style-permutations.page.tsx

```diff
@@ -0,0 +1,188 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import Tabs, { TabsProps } from '~components/tabs';
+
+import createPermutations from '../utils/permutations';
+import PermutationsView from '../utils/permutations-view';
+import ScreenshotArea from '../utils/screenshot-area';
+
+// No background, underline
+const style1 = {
+  tabs: {
+    backgroundColor: {
+      default: 'transparent',
+      hover: 'light-dark(rgba(59, 130, 246, 0.08), rgba(59, 130, 246, 0.15))',
+      active: 'transparent',
+      disabled: 'transparent',
+    },
+    borderColor: {
+      default: 'transparent',
+      hover: 'transparent',
+      active: 'transparent',
+      disabled: 'transparent',
+    },
+    borderWidth: '0px',
+    borderRadius: '6px',
+    color: {
+      default: 'light-dark(#64748b, #94a3b8)',
+      hover: 'light-dark(#1e293b, #e2e8f0)',
+      active: 'light-dark(#0f172a, #ffffff)',
+      disabled: 'light-dark(#cbd5e1, #475569)',
+    },
+    fontSize: '14px',
+    fontWeight: '500',
+    paddingBlock: '12px',
+    paddingInline: '20px',
+    focusRing: {
```

Comment:
I think `focusRing` color might need states, because it is displayed over the background color. So, for example, it might not be possible to find an accessible focus ring color that works over both default and active background colors

Reply:
For most components it appears outside the component itself so it's less of an issue as the "background" its displayed against isn't state-dependent. I guess the other alternative is to explore whether the ring could be moved outside the background here?

Reply:
Right, it's also displayed outside of the regular button component. That would make the API more streamlined across components, but I'm not sure if moving the ring will break something... I'll try to explore this further.